dizoo/slime_volley/envs/slime_volley_env.py

```diff
@@ -116,38 +119,24 @@ def reset(self):
         else:
             return obs
 
-    def info(self):
-        T = EnvElementInfo
-        return BaseEnvInfo(
```

Comment:
Was `BaseEnvInfo` not required any more when adding new environment?

Reply:
Yes, we remove it to facilitate the transfer of new env. And you can try use `DingEnvWrapper` to convert normal gym env to DI-engine format.